### PR TITLE
Create new API for attendance events and attendees

### PR DESCRIPTION
**Pull request by sardred**
Adds an endpoint for Attendees.
Updates a seemingly unused AttendeeSerializer and UserSerializer
Added (an easier) way to check if an attendee is on a waitlist.
The main magic here happens in the AttendViewSet that takes post requests from regme and handles the end user accordingly.

Sorry for the poorly organized commits. :disappointed:

### CODE REVIEW

**Comment by sardred** [comment]
This might not be done yet. We're not quite sure how we're going to get the events to begin with, so we might require some changes there if we can't find a clever way to filter our way to what we want.

**Comment by sardred** [comment]
I added filtering by attendance event. Everything should be complete now, and (hopefully) ready to merge.

**Comment by sardred** [comment]
Okay, THIS time it's ready.

File: apps/events/views.py [hunk 1]
```diff
@@ -319,7 +322,58 @@ class AttendanceEventViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin,
...
+        waitlist_approved = request.data.get('approved')
+        attendee = None
+
+        if username is not None:
+            try:
+                user = User.objects.get(username=username)
+                user.rfid = rfid
+                user.save()
+            except User.DoesNotExist:
+                return Response({'message': 'Brukernavn finnes ikke. Husk at det er brukernavn på Onlineweb! '
```
Comment (sklirg) [inline apps/events/views.py]: Could we use `online.ntnu.no` instead of `onlineweb`?

**Comment by sardred** [comment]
For real this time though.

**Pull request merged**